v3/lints/cabf_br/lint_invalid_ca_certificate_policies.go

```diff
@@ -0,0 +1,93 @@
+/*
+ * ZLint Copyright 2024 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/*
+ * Contributed by asantoni64@gmail.com
+ */
+
+package cabf_br
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+)
+
+func init() {
+	lint.RegisterCertificateLint(&lint.CertificateLint{
+		LintMetadata: lint.LintMetadata{
+			Name:          "e_invalid_ca_certificate_policies",
+			Description:   "Checks compliance with CABF section 7.1.2.10.5",
+			Citation:      "CABF BRs §7.1.2.10.5",
+			Source:        lint.CABFBaselineRequirements,
+			EffectiveDate: util.CABFBRs_2_0_0_Date,
+		},
+		Lint: NewInvalidCACertificatePolicies,
+	})
+}
+
+type invalidCACertificatePolicies struct{}
+
+func NewInvalidCACertificatePolicies() lint.LintInterface {
+	return &invalidCACertificatePolicies{}
+}
+
+func (l *invalidCACertificatePolicies) CheckApplies(c *x509.Certificate) bool {
+	return util.IsCACert(c) && !util.IsRootCA(c)
+}
+
+func (l *invalidCACertificatePolicies) Execute(c *x509.Certificate) *lint.LintResult {
+	if !util.IsExtInCert(c, util.CertPolicyOID) {
+		return &lint.LintResult{
+			Status:  lint.Error,
+			Details: "In a non-Root CA certificate, the CertificatePolicies extension is mandatory",
+		}
+	}
+
+	if len(c.PolicyIdentifiers) == 1 {
```

Comment:
If I'm reading this BR text correctly, it's not forbidden to have only the `A Reserved Certificate Policy Identifier`. One `MAY` have additional policy identifiers.

Reply:
From TLS BR 7.1.2.10.5:

> Regardless of the order of `PolicyInformation` values, the Certificate Policies extension MUST contain exactly one Reserved Certificate Policy Identifier.

https://github.com/cabforum/servercert/blob/main/docs/BR.md?plain=1#L2974

This will be relaxed for cross certificates as part of SC-79 (assuming it passes).